packages/gatsby-plugin-google-gtag/src/gatsby-ssr.js

```diff
@@ -6,6 +6,8 @@ exports.onRenderBody = (
   pluginOptions
 ) => {
   if (process.env.NODE_ENV !== `production`) return null
+  
+  pluginOptions.gtagConfig["send_page_view"] = false
```

Comment:
Can you add comment explaining why this is used? To prevent duplicate `pageview` events being emitted on initial load of page, because we send `pageview` in `onRouteUpdate` in `gatsby-browser.js` which also handles excluded pages config)

Reply:
I'm not sure right now if `gtagConfig` is required by plugin - but can there be scenario when this is `undefined` so maybe some sanitization to ensure that `gtagConfig` is an object:
```js
const gtagConfig = pluginOptions.gtagConfig || {}
```
and then use that new object?